### PR TITLE
add flattener for aws_rekognition_video_faces

### PR DESCRIPTION
**Pull request by sanzgiri**
modeled after your other aws flatteners, some of the features could be expanded further, but I kept them as is for consistency with other flatteners.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -499,6 +499,77 @@ def flatten_aws_rekognition_video_labels(self, run_options):
...
+                        details_obj['box'] = {'w': round(local_obj['BoundingBox']['Width'], 4), 
+                            'h': round(local_obj['BoundingBox']['Height'], 4),
+                            'l': round(local_obj['BoundingBox']['Left'], 4), 
+                            't': round(local_obj['BoundingBox']['Top'], 4) }
+                    score_frame = round(float(local_obj["Confidence"])/100, 4)
+
+                    list_items.append({"time_start": time_frame, "source_event": "image",
+                        "time_end": time_frame, "time_event": time_frame,
+                        "score": score_frame, "details": json.dumps(details_obj),
+                        "ageRange": local_obj["AgeRange"],
```
Comment (ezavesky) [inline main.py]: If you're okay with it, I'd like to pack these under the 'details_obj' json object to keep the final output with the same CSV-based schema, check out the recognition or label flatteners for more examples

Comment (ezavesky) [reply main.py]: also, I guess you woudl get this from the test input, but do we always have values for each of these items (e.g. "Beard" or "emotions")?

**Review by ezavesky — CHANGES_REQUESTED** [review]
proposed some changes, let me know what you think

File: main.py (hunk 1 above)
Comment (sanzgiri) [reply main.py]: yes, there is a  "Truee/False" or other value with each item and a confidence score. I can package them in the detailsObj for consistency.

**Comment by sanzgiri** [comment]
I updated the video_faces flattener, also added a person_tracking flattener

**Comment by ezavesky** [comment]
revise attributes in face to keep CSV simple

**Pull request merged**